README.md

```diff
@@ -13,71 +19,71 @@ Data must be provided as a list of maps, where each map is an observation, and t
 The mapping is specified with a map containing key-value pairs of variable names and aesthetics. Mappings must be provided for the x and y aesthetics; other aesthetics are optional, with fixed default values applied if a mapping is not specified. See the documentation for supported aesthetics for each geom.
 
 ```elixir
-    Examples.mtcars()
-    |> Plot.new(%{x: :wt, y: :mpg})
-    |> Plot.labs(title: "Basic Plot", x: "Weight (lbs)", y: "Miles Per Gallon")
-    |> Plot.geom_point()
-    |> Plot.plot()
+Examples.mtcars()
+|> Plot.new(%{x: :wt, y: :mpg})
+|> Plot.labs(title: "Basic Plot", x: "Weight (lbs)", y: "Miles Per Gallon")
+|> Plot.geom_point()
+|> Plot.plot()
 ```
-![](./examples/geom_point_basic.svg)
+![](./assets/geom_point_basic.svg)
```

Comment:
I could not figure out where to put these to have them show up in the docs; thanks for showing me how!

That said, I don't mind not having them in the docs; they kind of clutter them, I think. And they look pretty bad in dark mode. So I would prefer not to make this change.

Reply:
> I could not figure out where to put these to have them show up in the docs; thanks for showing me how!

:+1: 

> That said, I don't mind not having them in the docs; they kind of clutter them, I think. And they look pretty bad in dark mode. So I would prefer not to make this change.

To work around it, I've manually set background to white to all SVG assets and it looks something like this in dark mode.

![image](https://user-images.githubusercontent.com/134518/100538786-c2251900-326c-11eb-8db0-1875a80095c0.png)

Perhaps we can use this workaround for now until we can manually set a background color to generated SVG image? Maybe through `Plot.plot/1`?

What do you think?

Reply:
Thanks - I appreciate your effort here, but I just don't like having them in the docs this way. I think part of it is that ultimately I would like to basically copy the approach of the ggplot2 website, where for each type of geom, scale, etc., there are examples followed by graphics. For example, for `Plot.geom_point/3` you might see something [like this](https://ggplot2.tidyverse.org/reference/geom_point.html). At that point I will probably take all of these examples out of the readme altogether in favor of just one graphic showing the kinds of things you can do (similar to the [homepage of the demo site](http://www.pocketbookvote.com)), which I wouldn't want in the docs either as it is really just to pique interest as opposed to inform.

So I am eternally grateful for the education as to how to do this, but I would still prefer to keep these out of the readme in the docs.

Reply:
> Thanks - I appreciate your effort here, but I just don't like having them in the docs this way. I think part of it is that ultimately I would like to basically copy the approach of the ggplot2 website, where for each type of geom, scale, etc., there are examples followed by graphics. For example, for `Plot.geom_point/3` you might see something [like this](https://ggplot2.tidyverse.org/reference/geom_point.html). At that point I will probably take all of these examples out of the readme altogether in favor of just one graphic showing the kinds of things you can do (similar to the [homepage of the demo site](http://www.pocketbookvote.com)), which I wouldn't want in the docs either as its purpose is really just to pique interest as opposed to inform.

I see. I've reverted these assets change back to the original state, not showing in HTML doc.

> 
> So I am eternally grateful for the education as to how to do this, but I would still prefer to keep these out of the readme in the docs.

:smile: